data/gui/window/campaign_dialog.cfg

```diff
@@ -315,15 +315,21 @@
 
 					[option]
 						label = _ "Default RNG"
-						tooltip = _ "Reloading alters future combat outcomes"
+						tooltip = _ "Pure, unbiased randomness; the way Wesnoth is intended to be played.
+<span> </span>
+Example: if you strike four times with 50% accuracy, you’re most likely to hit twice and miss twice, but might hit four times, miss four times, or anywhere in between."
 					[/option]
 					[option]
 						label = _ "Predictable RNG"
-						tooltip = _ "Combat outcomes remain constant when reloading"
+						tooltip = _ "Identical to Default RNG, except loading a saved game will not change the outcome of an attack.
+<span> </span>
+Example: you strike four times and get lucky, hitting all four strikes. You then load an earlier save and make the same attack again. All four strikes will still hit."
 					[/option]
 					[option]
-						label = _ "Biased RNG (experimental)"
-						tooltip = _ "Combat outcomes are more in line with displayed probabilities and unaffected by reloading"
+						label = _ "Reduced RNG"
+						tooltip = _ "Attacks are much more consistent. Since humans plan more intelligently than Wesnoth’s AI, this tends to make the AI easier to outsmart.
+<span> </span>
+Example: if you strike four times with 50% accuracy, you will always hit twice and miss twice."
```

Comment:
Two strikes instead of four is slightly simpler, so IMO it'd be better to word it that way. Also not sure if it makes sense to mention the AI part.

Reply:
Changed four strikes to two.

AI part isn't worded great, but I included it to try and tell players _"yes it's easier, no that's not because RNG is biased."_ Open to suggestions for improvement.

Reply:
I'd tend towards not mentioning that at all just so players don't start choosing Reduced RNG because it's considered an easy mode setting rather than because they want to have RNG affect them less.

Reply:
That's fair, but I feel it's best to be transparent.  If a player wants an "easy mode" and the existing campaign difficulties don't suffice, I see no reason to try and hide the option from them.

Reply:
> "yes it's easier, no that's not because RNG is biased."

Well, it actually _is_ because it's biased though. It's biased towards providing the expected value (aka average) rather than an actual random result.

Reply:
Ah, I meant the default RNG. As in, it's not biased against the player.